Build failed in step 'Installing automake for build'

### Issue submission guidelines
See [detaled guidelines](http://crosstool-ng.github.io/support/). In short:
- Report crosstool-NG version
Relased 1.23.0 http://crosstool-ng.github.io/2017/04/19/release-1.23.0.html
- Report host OS and its version
Xubuntu 18.04, uname -a returns this: Linux DragonEnergy 4.15.0-20-generic #21-Ubuntu SMP Tue Apr 24 06:16:15 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
- Attach `.config` (unless the issue is not related to any configuration)
[confignothidden.txt](https://github.com/crosstool-ng/crosstool-ng/files/2018799/confignothidden.txt)
- Attach `build.log` (unless crosstool-NG fails before the build starts)
[build.log](https://github.com/crosstool-ng/crosstool-ng/files/2018795/build.log)


I am trying to compile C code on my x86 laptop that I can port to my raspberry pi zero w

Hi,
I'm facing issues with on ct-ng v1.9.3 with my OS as Mint 18.1
This is for 'arm-cortex_a8-linux-gnueabi'
When I run the './ct-ng build' command, it fails:
`
[INFO ]  Performing some trivial sanity checks
[INFO ]  Build started 20180627.171656
[INFO ]  Building environment variables
[EXTRA]  Preparing working directories
[EXTRA]  Installing user-supplied crosstool-NG configuration
[EXTRA]  =================================================================
[EXTRA]  Dumping internal crosstool-NG configuration
[EXTRA]    Building a toolchain for:
[EXTRA]      build  = x86_64-unknown-linux-gnu
[EXTRA]      host   = x86_64-unknown-linux-gnu
[EXTRA]      target = arm-cortex_a8-linux-gnueabi
[EXTRA]  Dumping internal crosstool-NG configuration: done in 0.07s (at 00:01)
[INFO ]  =================================================================
[INFO ]  Retrieving needed toolchain components' tarballs
[EXTRA]    Retrieving 'gmp-4.3.2'
[EXTRA]    Saving 'gmp-4.3.2.tar.bz2' to local storage
[EXTRA]    Retrieving 'mpfr-2.4.2'
[EXTRA]    Saving 'mpfr-2.4.2.tar.bz2' to local storage
[EXTRA]    Retrieving 'ppl-0.10.2'
[EXTRA]    Saving 'ppl-0.10.2.tar.bz2' to local storage
[EXTRA]    Retrieving 'cloog-ppl-0.15.9'
[ERROR]    Could not retrieve 'cloog-ppl-0.15.9'.
[02:32] / ct-ng:143: recipe for target 'build' failed
make: *** [build] Error 1
`
I checked the log, and realized that the web URLs for downloading many of the tarballs are outdated. How to overcome this problem? TIA.
[build.log](https://github.com/crosstool-ng/crosstool-ng/files/2141125/build.log)

--- a/packages/automake/1.11.6/0000-escape-left-brace.patch
+++ b/packages/automake/1.11.6/0000-escape-left-brace.patch
@@ -1,0 +1,27 @@
+commit 13f00eb4493c217269b76614759e452d8302955e
+Author: Paul Eggert <eggert@cs.ucla.edu>
+Date:   Thu Mar 31 16:35:29 2016 -0700
+
+    automake: port to Perl 5.22 and later
+    
+    Without this change, Perl 5.22 complains "Unescaped left brace in
+    regex is deprecated" and this is planned to become a hard error in
+    Perl 5.26.  See:
+    http://search.cpan.org/dist/perl-5.22.0/pod/perldelta.pod#A_literal_%22{%22_should_now_be_escaped_in_a_pattern
+    * bin/automake.in (substitute_ac_subst_variables): Escape left brace.
+
+---
+ automake.in |    2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+--- a/automake.in
++++ b/automake.in
+@@ -4156,7 +4156,7 @@
+ sub substitute_ac_subst_variables ($)
+ {
+   my ($text) = @_;
+-  $text =~ s/\${([^ \t=:+{}]+)}/&substitute_ac_subst_variables_worker ($1)/ge;
++  $text =~ s/\$[{]([^ \t=:+{}]+)}/&substitute_ac_subst_variables_worker ($1)/ge;
+   return $text;
+ }
+ 

--- a/packages/automake/1.14.1/0000-escape-left-brace.patch
+++ b/packages/automake/1.14.1/0000-escape-left-brace.patch
@@ -1,0 +1,27 @@
+commit 13f00eb4493c217269b76614759e452d8302955e
+Author: Paul Eggert <eggert@cs.ucla.edu>
+Date:   Thu Mar 31 16:35:29 2016 -0700
+
+    automake: port to Perl 5.22 and later
+    
+    Without this change, Perl 5.22 complains "Unescaped left brace in
+    regex is deprecated" and this is planned to become a hard error in
+    Perl 5.26.  See:
+    http://search.cpan.org/dist/perl-5.22.0/pod/perldelta.pod#A_literal_%22{%22_should_now_be_escaped_in_a_pattern
+    * bin/automake.in (substitute_ac_subst_variables): Escape left brace.
+
+---
+ bin/automake.in |    2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+--- a/bin/automake.in
++++ b/bin/automake.in
+@@ -3872,7 +3872,7 @@
+ sub substitute_ac_subst_variables
+ {
+   my ($text) = @_;
+-  $text =~ s/\${([^ \t=:+{}]+)}/substitute_ac_subst_variables_worker ($1)/ge;
++  $text =~ s/\$[{]([^ \t=:+{}]+)}/substitute_ac_subst_variables_worker ($1)/ge;
+   return $text;
+ }
+ 

